example/index.js

```diff
@@ -138,7 +130,7 @@ app.all("/", function(req, res, next) {
     next();
 });
 
-app.post("/", bodyParser.urlencoded());
+app.post("/", bodyParser.urlencoded({extended: false}));
```

Comment:
Revert to defaults otherwise readers may go off on a wild goose chase thinking this is a requirement of this module.

Reply:
Using the default configuration displays a warning on the console that the default will change in the next major version update.

Do you mean put "extended: true", or just leave it to the defaults and display the warning?